notebooks/getting_started/bq_dataframes_template.ipynb

```diff
@@ -0,0 +1,1848 @@
+{
```

Comment:
<p>Maybe add <code>--upgrade</code> to the install command since it seems from the internal chat thread thatsome folks might have older version installed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Makes sense, thanks. Done.